packages/carousel/src/react/controls.js

```diff
@@ -55,16 +62,21 @@ export function Control(props) {
   const handleClick = combineFns(isPrev ? prev : next, props.onClick)
 
   return (
-    <li>
-      <button
-        aria-label={isPrev ? 'previous' : 'next'}
-        {...styles.control(themeName, props)}
-        {...filterReactProps(props, { tagName: 'button' })}
-        {...(!visible && { hidden: true, tabIndex: -1 })}
-        onClick={handleClick}
-      >
-        <IconCaret aria-hidden size={iconSizes.medium} />
-      </button>
+    <li
```

Comment:
new focus style that uses halo. some props and elements got shuffled around to make it work